ext/README.md

```diff
@@ -3,6 +3,38 @@
 CEL extensions are a related set of constants, functions, macros, or other
 features which may not be covered by the core CEL spec.
 
+## Encoders
+
+Encoding utilies for marshalling data into standardized representations.
+
+### Base64.Decode
+
+Decodes bytes or a base64-encoded string to a sequence of bytes.
+
+This function will return an error if the UTF-8 string input is not
+base64-encoded.
+
+    base64.decode(<bytes>)  -> <bytes>
+    base64.decode(<string>) -> <bytes>
+
+Examples:
+
+    base64.decode(b'aGVsbG8=') // return b'hello'
+    base64.decode('aGVsbG8=')  // return b'hello'
+    base64.decode('aGVsbG8')   // error
+
+### Base64.Encode
+
+Encodes a bytes or string input to base64-encoded bytes.
+
+    base64.encode(<bytes>)  -> <bytes>
+    base64.encode(<string>) -> <bytes>
```

Comment:
Specify the the string is converted to bytes via utf8, then encoded.

Reply:
Done.